FrontEnd/src/components/Navigation.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+
+import { Row, Col } from 'react-materialize';
+
+///section css
+import componentCSS from './Navigation.scss';
+
+class Navigation extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			pages: [
```

Comment:
this probs. should be called "NaviationElement" or something along these lines, and use its own class